examples/natmod/README.md

```diff
@@ -1,33 +1,33 @@
 # Dynamic Native Modules
 
 Dynamic Native Modules are .mpy files that contain native machine code from a
-language other than Python. For more info see [the documentation]
-(https://docs.micropython.org/en/latest/develop/natmod.html).
+language other than Python. For more info see [the
+documentation](https://docs.micropython.org/en/latest/develop/natmod.html).
 
-This should not be confused with [User C Modules]
-(https://docs.micropython.org/en/latest/develop/cmodules.html) which are a
-mechanism to add additional out-of-tree modules into the firmware build.
+This should not be confused with [User C
+Modules](https://docs.micropython.org/en/latest/develop/cmodules.html) which are
+a mechanism to add additional out-of-tree modules into the firmware build.
 
 ## Examples
 
 This directory contains several examples of writing dynamic native modules, in
 two main categories:
 
-1.  Feature examples.
+1. Feature examples.
 
     * `features0` - A module containing a single "factorial" function which
       demonstrates working with integers.
 
     * `features1` - A module that demonstrates some common tasks:
-        - defining simple functions exposed to Python
-        - defining local, helper C functions
-        - defining constant integers and strings exposed to Python
-        - getting and creating integer objects
-        - creating Python lists
-        - raising exceptions
-        - allocating memory
-        - BSS and constant data (rodata)
-        - relocated pointers in rodata
+       * defining simple functions exposed to Python
+       * defining local, helper C functions
+       * defining constant integers and strings exposed to Python
+       * getting and creating integer objects
+       * creating Python lists
+       * raising exceptions
+       * allocating memory
+       * BSS and constant data (rodata)
+       * relocated pointers in rodata
```

Comment:
Would be good to stick with 4 space indentation.  That matches the rest of the project, and other .md files.

Does the linter also complain about * vs - bullet points?

Reply:
I'll roll it back to four spaces. It generates one warning [MD007](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md007---unordered-list-indentation) for the unordered list; it really wants 3 spaces. But it's a benign warning.

Yes, it raises [MD004](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md004---unordered-list-style) warnings if I use '-'. Reading the warning though, 'consistent' should be the default style and that I don't think we're breaking that rule; could be a false positive? If you prefer '*' I can revert that change.